components/attention/w-attention-arrow.vue

```diff
@@ -1,36 +1,43 @@
+<script setup>
+  import { computed } from 'vue'
+  import { props as attentionProps, opposites, rotation } from './attentionUtil.js'
+  import { attention as ccAttention } from '@warp-ds/component-classes'
+  
+  const props = defineProps({
+    ...attentionProps,
+    direction: String
+  })
+
+  const arrowDirectionClassname = (dir) => {
+    const direction = dir?.charAt(0).toUpperCase() + dir?.slice(1);
+
+    return `arrowDirection${direction}`;
+  }
```

Comment:
This might need to be reactive? 🤔 Perhaps better to do it as a computed as well as arrowDirection?